features/_usi/file.exclude

```diff
@@ -3,3 +3,6 @@
 /etc/systemd/system/ignition-disable.service
 /etc/repart.d/root.conf
 /usr/lib/tmpfiles.d/home.conf
+/opt/{,.}*
```

Comment:
2 questions:

what is the meaning of `{,.}*`? I see it is a regex but the comma dot is unclear to me. What is the intention?

What does excluding those files here do? What would happen if we did not do this?

Reply:
not a regex, just bash shorthand for `/opt/*` and `/opt/.*` since in bash `*` by default omits files starting with a dot

This excludes all files in `/opt`, `/home`, and `/root` but not the directories themselves. Since we are bind mounting over them there is no point in keeping any content inside them in the image